acceptance/tests/aix/nim_package_provider.rb

```diff
@@ -6,7 +6,7 @@
 
 teardown do
     test_apply('cdrecord', 'absent', '')
-    test_apply('bos.atm.atmle', 'absent', '')
+    test_apply('puppet.test.rte', 'absent', '')
```

Comment:
Would you mind noting somewhere in the test that this uses a fixture that is for all intents and purposes hidden and locked away somewhere on our nim master?